src/AddTodoForm.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
```

Comment:
Great work on the react assignment 1.3, here are a few things to keep working on:

Atomic commits, this time your commits looked better and they allowed me to understand what you were doing. I think they can still be more descriptive. For example, the last commit name was `final update` this is not very informative and it is not clear to me what the intent of these changes are. I would have preferred for you to use the instruction name for the commit, such as `store "NewTodo" in React state `.